content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -296,6 +296,59 @@ token available to the pod at a configurable file path, and refresh the token as
 
 The application is responsible for reloading the token when it rotates. Periodic reloading (e.g. once every 5 minutes) is sufficient for most usecases.
 
+## Service Account Issuer Discovery
+
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+
+{{< note >}}
+The Service Account Issuer Discovery feature is __alpha__ in 1.18 and enabled by
+setting the `ServiceAccountIssuerDiscovery` feature gate to `true` and then
+enabling the Service Account Token Projection feature as described above.
+
+Note that the issuer URL must comply with the
+[OIDC Discovery Spec](https://openid.net/specs/openid-connect-discovery-1_0.html). In
+practice, this means it must use the `https` scheme, and should serve an OpenID
+provider configuration at `{service-account-issuer}/.well-known/openid-configuration`.
+
+If the URL does not comply, the `ServiceAccountIssuerDiscovery` endpoints will
+not be registered, even if the feature is enabled.
+{{< /note >}}
+
+The Service Account Issuer Discovery feature enables federation of Kubernetes
+service account tokens issued by a cluster (the _identity provider_) with
+external systems (_relying parties_).
+
+When enabled, the Kubernetes API server will serve an OpenID Provider
```

Comment:
```suggestion
When enabled, the Kubernetes API server provides a public OpenID Provider
```

Reply:
Done. Also added clarification on RBAC, thanks for the reminder with your comment about "public."